packages/react-core/src/components/Page/examples/PageMainSectionVariations.tsx

```diff
@@ -64,22 +64,22 @@ export const PageMainSectionPadding: React.FunctionComponent = () => {
 
   return (
     <Page header={header} sidebar={sidebar}>
-      <PageSection type="subnav">
+      <PageSection type="subnav" aria-label="Horizontal subnavigation">
```

Comment:
For these ones, maybe instead having the label be "With [type] type" (so "With subnav type" and so on).